resources/var/tmp/resources/init.groovy.d/scripts/init030installplugins.groovy

```diff
@@ -55,6 +55,21 @@ for (def shortName : plugins){
   }
 }
 
+// Make sure CAS-Plugin version is at least 1.4.3 to work with Jenkins 2.150.2 and following
+def minimalCasPluginVersion = "1.4.3"
+def currentCasPlugin = jenkins.getPluginManager().getPlugin('cas-plugin');
+if (currentCasPlugin == null) {
+  println "CAS-Plugin is not installed; Installing plugin..."
+  updateCenter.getPlugin('cas-plugin').deploy(true).get();
+} else {
+  def currentCasPluginVersion = currentCasPlugin.getVersion();
+  def sortedPluginVersions = [minimalCasPluginVersion, currentCasPluginVersion].sort()
```

Comment:
This looks to complicated for a compare.
Furthermore what happens if the cas-plugin has the version 1.10.0 ?